### PR TITLE
MK-963 - add project link to dar detail page

### PR DESCRIPTION
**Pull request by meek0**
and used directive to hide said link if nghref route is not configured

### CODE REVIEW

File: src/access/views/data-access-request-view.html [hunk 1]
```diff
@@ -67,6 +67,15 @@
 
     <div class="clearfix"></div>
 
+    <div class="clearfix voffset2">
```
Comment (ymarcon) [inline src/access/views/data-access-request-view.html]: Why clearfix?

File: src/access/views/data-access-request-view.html [hunk 2]
```diff
@@ -67,6 +67,15 @@
...
+      <div ng-if="dataAccessRequest.project" class="pull-right">
```
Comment (ymarcon) [inline src/access/views/data-access-request-view.html]: What does it looks like in drupal?

**Comment by meek0** [comment]
Removed the clearfix.
I made a directive that hides the link if the ngHref route is not found but otherwise it will look the same on both mica admin and drupal.

**Comment by meek0** [comment]
updated to hide the link if user doesn't have permission.

**Pull request merged**